endpointmanager/pkg/jsonexport/jsonexport_integration_test.go

```diff
@@ -174,6 +176,7 @@ func Test_getHistory(t *testing.T) {
 		fhirURL: "thisurldoesntexist.com",
 		store:   store,
 		result:  resultCh3,
+		monthlyExport: false,
```

Comment:
Maybe have a test where monthlyExport is true? I guess it won't really be different from these other tests since the test database doesn't have a bunch of historic data to use, but at least just to check that it returns data as expected?